native/src/modules/layout/containers/HeaderContainer.js

```diff
@@ -67,8 +104,21 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsTy
     })
   }
   const peeking = isPeekingRoute(state, { routeCity: route.city })
+  const shareUrlFromScene = ownProps.scene.route.params?.shareUrl
+  const shareUrl: any = getShareUrl(state.cityContent,
```

Comment:
Thats not possible, because we set shareUrl to null in getShareUrl() line 50. Cause we need this to check if the share-Button should be shown.

We could do something link:
- adding a shareShouldBeDisplayed variable
- only passing shareUrl if shareShouldBeDisplayed is true
- check in Header if shareSouldBeDisplayed is true and shareUrl is available and then show share-Button

But I think it is ok if we just leave string | null.